src/Components/value-selector/index.js

```diff
@@ -0,0 +1,17 @@
+const ValueSelector = ({valueSelectorType}) => {
```

Comment:
Do you think it makes more sense to change to using a switch statement here:

switch (valueSelectorType) {
    case 'sound':
      content = <div>Sound</div>;
      break;
    case 'time':
      content = <div>Time</div>;
      break;
    default:
      content = <div>You did something wrong</div>;
      break;
  }

return content;

Reply:
Yeah i can switch it to a switch statement. Is there a functional difference, I thought it was a styling preference

Reply:
no it's definitely just a styling preference lol, but i do think it looks cleaner with switch. Whatever you prefer tho